src/solver/simulation/solver.h

```diff
@@ -167,8 +167,8 @@ class ISimulation : public Impl
     HydroManagement pHydroManagement;
     //! Hydro hot start
     bool pHydroHotStart;
-    //! The first set of parallel year(s) was already run ?
-    bool pFirstSetParallelWasRun;
+    //! The first set of parallel year(s) with a performed year was already run ?
+    bool pFirstSetParallelWithAPerformedYearWasRun;
```

Comment:
I would correct the english
it is a paralleSet rather than a SetParallel
`pFirstParalleSetWithAPerformedYearWasRun;`